python/cudf/cudf/core/multiindex.py

```diff
@@ -1106,10 +1093,12 @@ def __getitem__(self, index):
         match = self.take(index)
         if isinstance(index, slice):
             return match
-        result = []
-        for level, item in enumerate(match.codes):
-            result.append(match.levels[level][match.codes[item].iloc[0]])
-        return tuple(result)
+        if isinstance(index, int):
+            # we are indexing into a single row of the MultiIndex,
+            # return that row as a tuple:
+            return match.to_pandas()[0]
+        else:
+            return match
```

Comment:
This is an API break since previously we'd return a `tuple` whereas now we return ` MultiIndex`. Is that a bug fix, or do we need to signal that this is a breaking change?

Reply:
Based on the test I'm assuming that the change is to the desired behavior.

Reply:
Yeah the behavior remains consistent.